src/modules/auth/screens/AccessControlScreen.js

```diff
@@ -55,9 +56,13 @@ function AccessControlCard(props) {
 					<h3>
 						{t('AccessControlScreen|Access control')}
 					</h3>
+
 				</div>
 			</CardHeader>
 			<ul className="list-group list-group-flush">
+				<div style={{ height: '300px' }}>
```

Comment:
300px may look good on some certain screens, but on small one this would be too large, on large resolutions this may be too small.  Please test that (browsers allows you to test in various screens and resolutions >>> open the inspector of the browser and find something like responsive design mode). I would go with some dynamic option, like `className='h-80` or `className='h-100`

Reply:
please validate also in other cards. If you find out that your height setup is fine in various resolutions, then consider my comment as obsoleted.